### ISSUE TITLE
Problem login Porsche Connect - install integration

### ISSUE THREAD

**Issue by Rich-Kess** [CJNE/ha-porscheconnect#247]
Dear Mr., Mrs,

The Porsche Connect integration doesn't work anymore, is it not supported anymore?
In the past, for example after an update of HA, I discovered often that it didn't work. 
Then I deleted the integration, rebooted HA and tried to install it again.

When deleting Porsche Connect (and after restarting HA), I enter my email and password (from my Porsche), but then the system shows red icon with a ! connect, as if my login is incorrect, even though these are the correct credentials. I then have to log in to Porsche website again and change my password and then I can get the integration installed.

This is how it worked several times in the past year.
Only now this no longer seems to work, no matter what I try (have requested a new password from Porsche several times, but I keep having the same problem.
Just don't get the integration installed anymore.

Here are the details of what I am using;

My Porsche app (v11.24.24 build 322)
HA Core 2024.7.1
HA Supervisor 2024.06.2
HA OS 12.4
Frontend 20240705.0

Hope someone can help me with this, because i really like this integration.
Please let me know.

Greetings,

Richard

**Comment by simonbriers** [comment]
I have the exact same issue.

**Comment by Rich-Kess** [comment]
Anybody know who could fix this issue?

**Comment by barto64** [comment]
Please refer to: https://github.com/CJNE/ha-porscheconnect/issues/242

It is being worked on but will take time, so be patient ...

Not me (unfortunately as i do not have enough tech skills) but there is people in the community using their personal time to sort this out.

**Comment by Rich-Kess** [comment]
Ok great, thanks for your reply and have a nice weekend.

**Comment by ronniebach** [comment]
Any news on this issue ?

**Comment by Rich-Kess** [comment]
I haven't received any update since barto64 sent me his comment 2 weeks ago.

**Comment by midget1234** [comment]
Hi barto64 is this still something that is being worked on ? Do you know by whom ?

**Comment by barto64** [comment]
In #242 you will find additional info, but seem to be moving slowly.

**Comment by barto64** [comment]
And also on the only issue open under the underlying api this custom component use under CJNE/porscheconnectapi

**Comment by OlofBailey** [comment]
Anyone been able to make this work?

**Comment by 992tts** [comment]
Can't login and can't get it to work :-(

**Comment by fredriklj** [comment]
If you want to try out the ongoing effort to align with the new API, you need to check out and install the branch "new_api" of both this repo and pyporscheconnectapi.

Note that you will have to make sure the new_api branch of pyporscheconnectapi is installed in your hass environment before you start hass. It will not automatically pull a development branch and install it for you. How you proceeed with this depends on how you run hass (a development environment, a container, hassio or whatever).

**Comment by WensleyStock** [comment]
I suspect that the issues may well be  the same as the issues that other phone apps that use Porsche connect api have had issues with -
- the almost random occurrence of a captcha being presented as part of the re-authentication flow
- The weird behaviour of the Porsche backend servers that mean that some are cycled more frequently, which exacerbates point number 1
- The authentication token timeout which has been getting shorter and shorter meaning that re-authentication is generally required on a much shorter timeframe ( which seems to vary depending upon the backend server - see also point number 2). 
-there is a bug between the car and the servers that sometimes means that the backend thinks that the car is in privacy mode ( when it is not). 


Basically, Porsche have made a highly buggy and difficult to use backend and I suspect that it will not improve any time soon. 

We can only hope that they go down the apiKey route and get rid of the almost random captcha workflow.  I did here a rumour of an app development partner scheme but suspect that it may require paying into.
